docs/fleet/discovery.md

```diff
@@ -0,0 +1,15 @@
+title: Discovery Overview
+description: Overview of Discovery - Service and Logs Autodiscovery
+
+Discovery extends the capabilities of Fleet by automating the identification and categorization of services and collecting log data from various log sources. Below, you can explore some of the main benefits of Discovery:
+
+- Centralized view of all the discovered services and their current configurations within one or more Monitoring and Logs Integrations
+- Discovered services that can be automatically monitored through one of our [supported integrations](https://sematext.com/docs/integration/), eliminating the need for manual tracking and configuration
+- Automatic monitoring of newly discovered service instances, which is extremely useful in cluster and container environments
+- Automatic correlation between discovered services and logs using the [Connected Apps](https://sematext.com/docs/guide/connected-apps/) feature.
```

Comment:
Or maybe:

Automatic correlation between discovered services and logs using the [Connected Apps](https://sematext.com/docs/guide/connected-apps/) feature ==>
Automatic correlation between discovered and [connected services and logs](https://sematext.com/docs/guide/connected-apps/) using the [Split Screen](...) feature.